__tests__/app.test.js

```diff
@@ -298,6 +298,31 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
   
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("204: Deletes the given comment and responds with no content", () => {
+      return request(app)
+        .delete("/api/comments/1") 
+        .expect(204);
+    });
```

Comment:
Really nice just looking for the 204 🥇 We don't need a body here, there's no content!